debianstatic/postinst.in

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+FILE=/var/renasar/on-http/.skupack/{{name}}.tar.gz
+SKUFILE=/var/renasar/on-http/.skupack/{{name}}.sku
+CODE=`curl -s -X POST --data-binary @${FILE} -o ${SKUFILE} -w '%{http_code}' localhost:8080/api/common/skus/pack`
```

Comment:
Yep, that's what happens if the API is offline.  The API has to be up to service the request to create the SKU from the rules definition and process the nodes against it.

The else prompts the user to retry registering:
+  echo SKU Pack registration failed. Please retry registering
 +  echo the pack at: ${FILE}